packages/flutter/lib/src/gestures/events.dart

```diff
@@ -218,6 +231,7 @@ class PointerAddedEvent extends PointerEvent {
   );
 }
 
+/// The pointer has been removed from the device.
```

Comment:
"The device is no longer tracking the pointer."

Also maybe add "// TODO(ianh): actually use this class" to added and removed. :-)